src/search/mod.rs

```diff
@@ -74,6 +74,7 @@ pub struct UploadSearchProject {
     pub title: String,
     pub description: String,
     pub categories: Vec<String>,
+    pub display_categories: Vec<String>,
```

Comment:
This seems really confusingly named. Instead of making secondary categories `categories` and primary categories `display_categories`, can't we just have `categories` and `secondary_categories`?

Reply:
categories contains both additional and primary, so people can search by additional